.prettierignore

```diff
@@ -1,7 +1,6 @@
 .github/**
 demos/**
 dist/**
-lib/types/**
```

Comment:
did this end up changing the style of the protos at all?

Reply:
It changed the styles of the generated TS files which caused the diff to be huge. The [generate_types.sh](https://github.com/lightninglabs/lnc-web/blob/main/scripts/generate_types.sh#L181) script runs `prettier` on the generated files so that the formatting stays consistent after each version update and the diff only shows what fields have been added/removed. Personally, I think this is better, but I'm open to hear your thoughts.

Reply:
Thanks for the explanation. I think this is the right call.